src/data/waypoint.json

```diff
@@ -59,7 +59,7 @@
 			"type": "inbound"
 		}
 	],
-	"basePaths": ["commands", "docs", "plugins", "tutorials", "downloads"],
+	"basePaths": ["commands", "docs", "integrations", "tutorials", "downloads"],
```

Comment:
I think `basePaths` is [being deprecated](https://github.com/hashicorp/dev-portal/blob/main/src/data/README.md#devdot-properties), and is also used for old link rewrites, so I think we might want to maintain `"plugins"` in this case 🤔

Reply:
Ah!  Okay, yep I'll return this back to normal.

FWIW, I didn't observe any changes when I switched this so we're definitely safe to go back.

Reply:
yeah tbh i'm not sure around impact! I think it `basePaths` would mostly apply to automatically rewriting any links to `waypointproject.io/plugins` within content... but we've since rewritten those types of links at source thanks to Ashlee & Kendall's efforts... and with #1600 & #1602 we rewrite plugins links in the newer format so this might be a non-issue either 🤷